src/plugins/intel_cpu/tests/functional/utils/cpu_test_utils.cpp

```diff
@@ -463,4 +464,44 @@ void CheckNumberOfNodesWithType(const ov::CompiledModel& compiledModel,
                                 size_t expectedCount) {
     CheckNumberOfNodesWithTypes(compiledModel, {nodeType}, expectedCount);
 }
+
+
+ov::element::Type
+CPUTestsBase::get_default_imp_precision_type(const ov::element::Type& in_type,
+                                             const ov::AnyMap& configuration) {
+#if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
+    return in_type;
+#endif
+#if defined(OPENVINO_ARCH_RISCV64)
+    return in_type;
+#endif
+#if defined(OPENVINO_ARCH_X86_64)
+    const std::string key = ov::hint::inference_precision.name();
+    const std::string KEY_ENFORCE_BF16 = "ENFORCE_BF16";
+    // if is not float
+    if (in_type != ov::element::f16 && in_type != ov::element::f32 && in_type != ov::element::bf16) {
+        return in_type;
+    }
```

Comment:
`ov::element::Type` has `is_real()` method.

Reply:
ok, will update it

Reply:
already updated it